docs2/Doxyfile

```diff
@@ -5,16 +5,54 @@ PROJECT_LOGO = ../data/CATChem_logo_h55.png
 
 OPTIMIZE_FOR_FORTRAN = YES
 
+CREATE_SUBDIRS         = NO
+CREATE_SUBDIRS_LEVEL   = 8
+
+ALLOW_UNICODE_NAMES    = NO
+OUTPUT_LANGUAGE        = English
+BRIEF_MEMBER_DESC      = YES
+MARKDOWN_SUPPORT       = YES
+
 INPUT = start.md
 INPUT += ../src/core
 INPUT += ../src/process/dust
 INPUT += ../src/process/seasalt
 USE_MDFILE_AS_MAINPAGE = start.md
-HTML_EXTRA_STYLESHEET = ../docs2/static/doxygen-awesome.css
+HTML_EXTRA_STYLESHEET = ../docs2/static/doxygen-awesome.css \
+						../docs2/static/doxygen-awesome-sidebar-only.css
```

Comment:
Can you align this with the previous line? Also shouldn't just `static/` work, instead of `../docs2/static/`?

Reply:
this is aligned in my editor.  Maybe it's a tab vs space issue?